mmrazor/engine/runner/utils/check.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+from typing import Optional, Tuple
+
+import torch.nn as nn
+from mmdet.models.detectors import BaseDetector
```

Comment:
Try to use PlaceHolder for downstream repos.

Reply:
Done.